Added Animation, Tilemaps, and many more (#24)
twolivesleft/Codea4-Docs: docs/source/api/physics2d.rst

   .. lua:attribute:: body: physics2d.body

      The body this collider belongs to
      
      .. helptext:: get the body this collider belongs to

   .. lua:method:: collide(otherCollider)

>       Checks the collision between two colliers: this one and another collider and gives infomation about it

Reviewer: ```suggestion
      Checks the collision between this collider and another collider, and returns information about the collision.
```

      :param otherCollider: The other collider to collide with
      :type otherCollider: collider

      :return: ``didCollide[, point, normal, penetration]`` - `didCollide` is whether the collision happened
      :rtype: boolean[, vec2, vec2, number]
      

>       .. helptext:: allows users to do check collision between two colliders

Reviewer: ```suggestion
      .. helptext:: check for a collision with another collider
```

   .. lua:method:: overlap(otherCollider)

>       Checks overlapping between two colliers: this one and another collider

Reviewer: ```suggestion
      Checks if the collider overlaps with another collider.
```

      :param otherCollider: The other collider to overlap with
      :type otherCollider: collider

      :return: Checks whether the two colliders are overlapping
      :rtype: boolean
      

>       .. helptext:: allows users to do check if two colliders are overlapping

Reviewer: ```suggestion
      .. helptext:: check for an overlap with another collider
```

.. lua:class:: motor: joint

   *Not implemented yet*         

Settings
########

.. lua:class:: settings

   .. lua:attribute:: debugDraw: boolean

>       Draws physics objects in the scene

Reviewer: ```suggestion
      Draws physics objects in the scene.
```

   .. lua:attribute:: gravity: vec2
      

>       Changes the gravity of the physics world

Reviewer: ```suggestion
      Changes the gravity of the physics world.
```

   .. lua:attribute:: velocityIterations: number

   .. lua:attribute:: positionIterations: number

   .. lua:attribute:: paused: boolean

>       Whether you want to paused the physics in a scene

Reviewer: ```suggestion
      Get or set whether the physics simulation is currently paused. 
```
